lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop check for usages of not called on a method when an
+      # inverse of that method can be used instead
+      #
+      # @example
+      #   # bad
+      #   !foo.none?
+      #   !foo.any? { |f| f.even? }
+      #   !foo.blank?
+      #
+      #   # good
+      #   foo.none?
+      #   foo.blank?
+      #   foo.any? { |f| f.even? }
+      class InverseMethods < Cop
+        MSG = 'Use `%s` instead of `!.%s`.'.freeze
+
+        RUBY_METHODS = {
```

Comment:
I had thought about this briefly. I will update this. Since this is going to be configurable, I guess there should be a single configuration for all methods instead of having them split by Ruby and ActiveSupport. I figure we can list methods such as `blank?` and have them commented out by default. What are you thoughts/recommendations on this? 

```yml
Style/InverseMethods:
  InverseMethods:
    :none?: :any?
    # :blank?: :present?
```